docs/branding.md

```diff
@@ -19,3 +19,20 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
 	}
 }
 ```
+
+## Custom admin colour scheme
```

Comment:
We're using title casing throughout so need to capitalise these words

Reply:
But `## Login screen` above is in low-case. I copied that. Must be missing something.
Do you mean `Custom Admin Colour Scheme`?

Reply:
Right ok, that's a mistake then sorry! We need to go through and clarify all that and make it consistent as part of a general documentation sweep